python/lsst/cp/pipe/ptc.py

```diff
@@ -411,7 +411,7 @@ def runDataRef(self, dataRef, visitPairs):
                 doRealSpace = self.config.covAstierRealSpace
                 muDiff, varDiff, covAstier = self.measureMeanVarCov(exp1, exp2, region=amp.getBBox(),
                                                                     covAstierRealSpace=doRealSpace)
-                if np.isnan(muDiff) or np.isnan(varDiff) or np.isnan(covAstier):
```

Comment:
Do we not care if the covariance is populated?  If this catches a failure, can we return `None` instead of the matrix, so we can check that case?